sdk/identity/identity/samples/AzureIdentityExamples.md

```diff
@@ -0,0 +1,358 @@
+# Table of contents
```

Comment:
@sadasant Is there anything in here that does not apply to v1 of Identity?

Reply:
DeviceCodeCredential doesn't accept direct parameters anymore. So, this file here is fine, but not this one: https://github.com/Azure/azure-sdk-for-js/pull/14646/files/99edb217ed79552fc329a9aa981f72fdeafa88c9#r606418739 - I'm still looking to see if I can spot anything else.

Reply:
I couldn't find anything else, so this file looks good to me!